charts/descheduler/values.yaml

```diff
@@ -22,6 +22,16 @@ resources:
   #   cpu: 100m
   #   memory: 128Mi
 
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+  privileged: false
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1000
```

Comment:
should we remove the `USER 1000` from the Dockerfile?

Reply:
that's a good question, let's discuss here, i check the the file https://github.com/kubernetes-sigs/descheduler/blob/master/kubernetes/deployment/deployment.yaml#L48 it does not have runasuser, so if we remove from the dockerfile, then i will add runasuser into the deployment/deployment.yaml .

Reply:
That makes sense. I personally like the idea of controlling the `runAsUser` in the manifest, if possible.

Reply:
if so, then i will try to remove runasuser from dockerfile, and put it into the manifest.

Reply:
@damemi thoughts? Is it too big of a breaking change to do that?

Reply:
does setting `runAsUser` override what's in the Dockerfile? The problem I can think of is that our public images are built and published with that, so it could break for users who aren't using our helm chart or default deployment manifests.

So if `runAsUser` overrides the Dockerfile, then I don't think we should remove it from the Dockerfile.

Reply:
Yes, it overrides it

Reply:
Let's leave it as-is for now and unblock this PR